nima_core/consciousness_workspace.py

```
@@ -0,0 +1,726 @@
+def demo_consciousness_workspace():
+    """Run a simple cognitive cycle with a few agents."""
+    print("=" * 70)
+    print("GLOBAL WORKSPACE + VSA CONSCIOUSNESS PROTOTYPE")
+    print("=" * 70)
+    
+    # Create workspace
+    ws = GlobalWorkspace(capacity=4)
+    
+    # Set initial affect
+    ws.affective_core.set_affect(Affects.SEEKING)
+    print(f"\n🧠 Affective state: SEEKING (exploratory)")
```

Ruff (0.15.0)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)